### DISCUSSION TITLE
PBPK Model

### DISCUSSION THREAD

**Discussion by pyrokinesis244** [Open-Systems-Pharmacology/Forum#1503] [Models] [answered]
Does the PK-Sim app have a Sublingual PBPK model included?

If yes, I will sincerely appreciate pointers to where the file is located. 

thanks in advance

**Comment by StephanSchaller** [comment, marked as answer]
NO, this is not natively included in PK-Sim and would require either use of a surrogate absorption path in PK-Sim or implementation through MoBi.

**Reply by cherryman-bobby** [reply]
hello stephan, Do you know any sources that use any of these methods? I would like to but I don't know how to even start looking at the problem as I have no experience in programming or physiology

**Reply by StephanSchaller** [reply]
Not that I am aware of.
But any experienced MoBi user can set this up in a short time based on a published model. You can learn it on our eLearning platform learn.esqlabs.com ... or work with a MoBi expert :-)
